src/display.cpp

```diff
@@ -2721,7 +2721,9 @@ void display::draw(bool update,bool force) {
 
 	if (dirty_) {
 		dirty_ = false;
+		screen_.lock_flips(true);
```

Comment:
Fixed in 
https://github.com/wesnoth/wesnoth/pull/697/commits/aa58d83b4500bd697ef1efcea1406fe5f9318322